src/services/email.service.js

```diff
@@ -0,0 +1,65 @@
+const nodemailer = require('nodemailer');
+
+const transporter = nodemailer.createTransport({
+  host: process.env.SMTP_HOST,
+  port: process.env.SMTP_PORT,
+  secure: true,
+  auth: {
+    user: process.env.SMTP_USER,
+    pass: process.env.SMTP_PASSWORD,
+  },
+});
+
+function send({ email, subject, html }) {
+  return transporter.sendMail({
+    from: 'Auth API',
+    to: email,
+    subject,
+    text: '',
+    html,
+  });
+}
+
+function sendActivationLink(email, token) {
+  const link = `${process.env.CLIENT_URL}/activate/${token}`;
+
+  return send({
+    email,
+    subject: 'Account activation',
+    html: `
+      <h1>Account activation</h1>
+      <a href="${link}">${link}</a>
+    `,
+  });
+}
+
+function sendReset(email, token) {
+  const href = `${process.env.CLIENT_HOST}/resetPassword/${email}/${token}`;
+
+  const html = `
+    <h1>Reset password</h1>
+    <a href="${href}">${href}</a>
+  `;
+
+  return send({ email, html, subject: 'Reset' });
+}
+
+function sendConfirmation(email, token) {
+  const href = `${process.env.CLIENT_HOST}/confirmation/${email}/${token}`;
```

Comment:
Inconsistent environment variable: `process.env.CLIENT_HOST` is used here, while `process.env.CLIENT_URL` is used in `sendActivationLink`. Ensure consistency in naming to avoid confusion or errors. Consider using `process.env.CLIENT_URL` for all URL constructions if they refer to the same client application.